.env.example

```
@@ -1,41 +1,4 @@
-# DSP/外部ツール
-RUBBERBAND_BIN=rubberband
-FEATURE_MICROADJUST=true
-FEATURE_TEMPO_DTW=true
-FEATURE_PITCH_ONE_NOTE_AUTO=true
-
-# Harmony
-HARMONY_OPTIONS=up_m3,down_m3,perfect_5th
-HARMONY_COST_LITE_C=0.5
-HARMONY_COST_STD_C=0
-HARMONY_COST_CREATOR_C=0
+﻿# examples (ダミー／絶対に本物は書かない)
```

dotenv-linter (3.3.0)

[warning] [IncorrectDelimiter] The ﻿# examples (ダミー／絶対に本物は書かない) key has incorrect delimiter

(IncorrectDelimiter)